addons/compat_rf/compat_rf_realisticnames/stringtable.xml

```diff
@@ -4,214 +4,267 @@
+            <Korean>ASh-12 LR (도심)</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ash12_lr_wood_Name">
             <English>ASh-12 LR (Woodland)</English>
             <Japanese>ASh-12 LR (森林迷彩)</Japanese>
+            <Korean>ASh-12 LR (수풀)</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_heli_light_03_Name">
             <English>AW159 Wildcat ASW</English>
             <Japanese>AW159 ワイルドキャット ASW</Japanese>
+            <Korean>AW159 와일드캣 ASW</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_heli_light_03_unarmed_Name">
             <English>AW159 Wildcat ASW (Unarmed)</English>
             <Japanese>AW159 ワイルドキャット ASW (非武装)</Japanese>
+            <Korean>AW159 와일드캣 ASW (비무장)</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ec_01_base_Name">
             <English>H225 Super Puma (Transport)</English>
             <Japanese>H225 シュペル ピューマ (輸送型)</Japanese>
+            <Korean>H225 슈퍼 퓨마 (비무장)</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ec_01_civ_base_Name">
             <English>H225 Super Puma (Civilian)</English>
             <Japanese>H225 シュペル ピューマ (民生型)</Japanese>
+            <Korean>H225 슈퍼 퓨마 (비무장)</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ec_01a_base_Name">
             <English>H215 Super Puma (Transport)</English>
             <Japanese>H215 シュペル ピューマ (輸送型)</Japanese>
+            <Korean>H215 슈퍼 퓨마 (비무장)</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ec_01a_civ_base_Name">
             <English>H215 Super Puma (Civilian)</English>
             <Japanese>H215 シュペル ピューマ (民生型)</Japanese>
+            <Korean>H215 슈퍼 퓨마 (비무장)</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ec_01a_military_base_Name">
             <English>H215 Super Puma (Unarmed)</English>
             <Japanese>H215 シュペル ピューマ (非武装)</Japanese>
+            <Korean>H215 슈퍼 퓨마 (비무장)</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ec_02_base_Name">
             <English>H225M Super Cougar SOCAT</English>
             <Japanese>H225M シュペル クーガー SOCAT</Japanese>
+            <Korean>H225M 슈퍼 쿠거 SOCAT</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ec_02_nato_Name">
             <English>H225M Super Cougar SOCAT</English>
             <Japanese>H225M シュペル クーガー SOCAT</Japanese>
+            <Korean>H225M 슈퍼 쿠거 SOCAT</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ec_03_base_Name">
             <English>H225M Super Cougar</English>
             <Japanese>H225M シュペル クーガー</Japanese>
+            <Korean>H225M 슈퍼 쿠거</Korean>
         </Key>
         <Key ID="STR_ACE_Compat_RF_RealisticNames_ec_04_base_Name">
             <English>H225 Super Puma SAR</English>
             <Japanese>H225 シュペル ピューマ 捜索救難型</Japanese>
+            <Korean>H225 슈퍼 퓨마 SAR</Korean>
```

Comment:
I think I fixed merge conflicts from new translations inhttps://github.com/acemod/ACE3/pull/10071
if there are any mistakes it will be if I mixed up the super puma/cougars